blocks/block_store.go

```diff
@@ -101,6 +102,10 @@ func (store *BlockStore) AddFile(reader *sequencefile.Reader) error {
 	assumedFor := 0
 
 	for reader.Scan() {
+		if throttle != 0 {
```

Comment:
tbc if `throttle_loads` is unset in the config, it defaults to a duration of `0` rather than defaulting to `nil`. I assume that's what's happening but didn't see anything in `toml` about defaults nor in this PR.